### DISCUSSION TITLE
Tags page as more symbolic

### DISCUSSION THREAD

**Discussion by dougie-r2** [1bl4z3r/hermit-V2#67] [Ideas]
I'd like to create my Tags page as the following link. [https://lilianweng.github.io/tags/](https://lilianweng.github.io/tags/)
Is this supported?
I am absolute new to these HTML and CSS things. I tried by myself with the help of chatGPT.
However, Tags page looks like Posts page. I end up writing here for help. haha

I like this theme and I just want to add tags feature on it. thx! 👍

**Comment by 1bl4z3r** [comment]
Hi @dougie-r2,

Firstly I never knew that there is a discussion panel in GitHub repos 😅

Secondly, it is absolutely supported. I mean it is not supported OOTB, you'll have to smack your head against it a couple of times to make it work (let's be honest, apart from hugo devs, everyone is using ....feelings....)

What you can do is take a look at list.html. It is responsible for anything that has a list (posts, tags, categories). Read it through. I am quite certain you will question your entire existence while going through it. Don't fret. Post your queries here. I will try my best to answer.

At the very end, if you wish to add your creation in this theme, put a variable in config to toggle legacy view and "new view (by dougie😉)"

**Reply by dougie-r2** [reply]
Thx for the reply !
I thought it would take some times to get the answer, but I am the one who reply late.
As you mentioned, I will look thoroughly list.html.

Thanks for creating this theme again ! :)

**Reply by 1bl4z3r** [reply]
Calcification. I didn't create this theme. I am just a maintainer, as og decided to not to support it 😤. 

Looking forward to be an audience of your creativity 😀 

Live long & Prosper

**Reply by 1bl4z3r** [reply]
Poppin in to say Hi👋

**Reply by 1bl4z3r** [reply]
Hi @dougie-r2,

Any updates on your venture?
